src/Features/Core/Portable/Shared/Extensions/ISymbolExtensions_2.cs

```diff
@@ -176,49 +176,8 @@ public static Glyph GetGlyph(this ISymbol symbol)
-                IMethodSymbol method => GetMethodDocumentation(method, compilation, cancellationToken),
-                IAliasSymbol alias => alias.Target.GetDocumentationComment(compilation, expandIncludes: true, expandInheritdoc: true, cancellationToken: cancellationToken).SummaryText,
-                _ => symbol.GetDocumentationComment(compilation, expandIncludes: true, expandInheritdoc: true, cancellationToken: cancellationToken).SummaryText,
-            };
-
-        private static string GetParameterDocumentation(IParameterSymbol parameter, Compilation compilation, CancellationToken cancellationToken)
-        {
-            var containingSymbol = parameter.ContainingSymbol;
-            if (containingSymbol.ContainingSymbol.IsDelegateType() && containingSymbol is IMethodSymbol methodSymbol)
-            {
-                // There are two ways to invoke a delegate that we care about here: the Invoke()/BeginInvoke() methods. (Direct invocation is equivalent to an Invoke() call.)
-                // DynamicInvoke() takes an object array, and EndInvoke() takes a System.IAsyncResult, so we can (and should) ignore those here.
-
-                var symbolName = methodSymbol.Name;
-                if (symbolName == WellKnownMemberNames.DelegateBeginInvokeName && parameter.Ordinal >= (methodSymbol.Parameters.Length - 2))
-                {
-                    // Return null (similar to DocumentationComment.GetParameterText()) for the last two implicit parameters (usually called "callback" and "@object").
-                    // We can't rely on those names because they might be renamed to avoid collision with a user-defined delegate parameter of the same name,
-                    // and we have to treat them separately, because a user might add e.g. a '<param name="callback">' tag to the delegate, which would be displayed in Signature Help for that implicit parameter.
-                    return null;
-                }
-
-                if (symbolName == WellKnownMemberNames.DelegateInvokeName || symbolName == WellKnownMemberNames.DelegateBeginInvokeName)
-                {
-                    // We know that containingSymbol is the [Begin]Invoke() method of a delegate type, so we need to go up a level and take the method's containing symbol (i.e. the delegate), which contains the documentation.
-                    containingSymbol = containingSymbol.ContainingSymbol;
-                }
-            }
-
-            // Get the comments from the original definition of the containing symbol.
-            return containingSymbol.OriginalDefinition.GetDocumentationComment(compilation, expandIncludes: true, expandInheritdoc: true, cancellationToken: cancellationToken).GetParameterText(parameter.Name);
+            return formatter.Format(symbol, semanticModel, position, cancellationToken, CrefFormat)
+                ?? SpecializedCollections.EmptyEnumerable<TaggedText>();
```

Comment:
do we need the `??` bit?  can we not just make `formatter.Format` always return a non-null value?

Reply:
probably. i wanted to keep the changes minimal

Reply:
done.